addons/mail/static/src/owl/components/activity/activity.xml

```diff
@@ -0,0 +1,147 @@
+                            <dd class="o_Activity_detailsType">
+                                <t t-esc="storeProps.activity.activityTypeDisplayName"/>
+                            </dd>
+                            <dt>Created on</dt>
+                            <dd class="o_Activity_detailsCreation">
+                                <t t-esc="formattedCreateDatetime"/>
+                                by
+                                <img t-att-src="creatorAvatarSource" height="18" width="18"
+                                    class="o_object_fit_cover rounded-circle mr4"
+                                    t-att-title="storeProps.activity.creatorDisplayName"
+                                    t-att-alt="storeProps.activity.creatorDisplayName"/>
+                                <span class="o_Activity_detailsCreator">
+                                    <t t-esc="storeProps.activity.creatorDisplayName"/>
+                                </span>
+                            </dd>
+                            <dt>Assigned to</dt>
+                            <dd class="o_Activity_detailsAssignation">
+                                <img t-att-src="assignedAvatarSource" height="18" width="18"
+                                     class="o_Activity_detailsAssignationUserAvatar rounded-circle"
+                                     t-att-title="storeProps.activity.userDisplayName"
+                                     t-att-alt="storeProps.activity.userDisplayName"/>
+                                <t t-esc="storeProps.activity.userDisplayName"/>
+                                <em>, due on </em>
+                                <span t-att-class="'o_Activity_color_'+storeProps.activity.activityState">
+                                    <t t-esc="formattedDeadlineDate"/>
+                                </span>
+                            </dd>
+                        </dl>
+                    </div>
+
+                    <div class="o_Activity_note">
+                        <t t-raw="storeProps.activity.note"/>
+                    </div>
+
+                    <t t-if="storeProps.activity.mailTemplates and storeProps.activity.mailTemplates.length > 0">
+                        <div class="o_Activity_mailTemplates">
+                            <t t-foreach="storeProps.activity.mailTemplates" t-as="mailTemplate" t-key="mailTemplate">
+                                <div class="o_Activity_mailTemplate">
+                                    <i class="fa fa-envelope-o" aria-label="Mail" title="Mail" role="img"/>
+                                    <span class="o_Activity_mailTemplate_name" t-esc="mailTemplate.name"/>
+                                    <span>:</span>
+                                    <button class="o_Activity_mailTemplate_button o_Activity_mailTemplate_preview btn btn-link"
+                                            t-att-data-mail-template-id="mailTemplate.id"
```

Comment:
as said before, business code should not rely on `dataset` with OWL, and instead make good use of sub-components instead